### DISCUSSION TITLE
Logging doesn't work

### DISCUSSION THREAD

**Discussion by morenostef** [stride3d/stride#2424] [Newcomers]
Hi,i've just installed stride 4.2. When i use c#, logging doesn't seem to work. No window opens to print log messages.
Visual studio 2022 is in debug mode. Thanks!

**Comment by NicusorN5** [comment]
How are you printing the message?

Are you using `Debug.Log`? `Console.Print`? Debug.Log outputs to a `Output` window in VS2022. it doesn't create a separate console window.

**Reply by morenostef** [reply]
Console.WriteLine opens a logging window if i run the game from VS2022.
Using Log.Warning,i can log from Game Studio,as @IXLLEGACYIXL  wrote.
Thank you anyway,now i can log everywhere :-D !!!

**Comment by IXLLEGACYIXL** [comment]
at the bottom https://doc.stride3d.net/latest/en/manual/stride-for-unity-developers/index.html
`Log.Warning("hello");` and it will open a logging window

**Reply by morenostef** [reply]
Thank you,now it works :-)
